.ci/lib.sh

```diff
@@ -417,3 +417,24 @@ get_pr_changed_file_details()
 {
 	get_pr_changed_file_details_full | grep -v "vendor/"
 }
+
+# Receives as list of files and/or, build a sha256sum from their contents
+sha256sum_from_files() {
+	# TODO: implement me!
+	echo "zzzzzzzzzzzzzzzzz"
+
+}
+
+gen_qemu_files_sha256sum() {
+	local pkg_dir="${kata_repo_dir}/tools/packaging"
+	# Files and dirs
+	files="${pkg_dir}/qemu \
```

Comment:
I think some of the scripts we use do `source` to other  scripts , even if we can not control what is sourced, it is still  a better approach than a number (that we need to modify manually) and I like the simplicity of the initial idea.

Reply:
Thanks for the feedback @jcvenegas . Still today I plan to send a working version of this PR.